openspec/changes/archive/2026-03-06-feat-cli-score-display/tasks.md

```
@@ -0,0 +1,13 @@
+## 1. 实现
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)